internal/indexer/definitions/btn.yaml

```diff
@@ -96,10 +96,12 @@ irc:
           - preTime
       - test:
           - "[ https://XXXXXXXXX/torrents.php?id=7338 / https://XXXXXXXXX/torrents.php?action=download&id=9116 ]"
-        pattern: \[ .* \/ (https?:\/\/.*\/).+id=(\d+) \]
+        pattern: \[ (https?:\/\/.*\/).+id=(\d+) \/ https?:\/\/.*\/.+id=(\d+) \]
         vars:
           - baseUrl
+          - groupId
           - torrentId
 
     match:
+      infourl: "/torrents.php?id={{ .groupId }}"
```

Comment:
We could extend the infourl here to make it open the info of the actual torrent instead of just opening the group here...
`/torrents.php?id={{ .groupId }}&torrentid={{ .torrentId }}`
Don't know if you care about that...